CONTRIBUTING.md

```diff
@@ -46,12 +55,18 @@ OpenStreetMap (and StreetComplete) works, because of a diverse community all ove
 If you experience problems, crashes or a quest is not clear to you, feel free to open an issue for that. Remember to open one issue _for one matter_, so do not open one issue "I have found several problems", but one for each problem.
 
 1. If you have questions, remember to [read the FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ) first.
-2. Look [whether your issue has already been reported](https://github.com/westnordost/StreetComplete/issues).
-3. Now you can open a new issue. Give it a meaningful title and describe the problem. Do mention the StreetComplete and Android version, if it is a technical bug.
+2. Remember to report map style issues [in the appropriate repository](#issues-of-dependencies).
+3. Look [whether your issue has already been reported](https://github.com/westnordost/StreetComplete/issues).
+4. Now you can open a new issue. Give it a meaningful title and describe the problem. Do mention the StreetComplete and Android version, if it is a technical bug.
 
-### Map style issues
+### Issues of dependencies
 
-Note that the map style used by StreetComplete is developed as a [separate project](https://github.com/ENT8R/streetcomplete-mapstyle), so please report issues related to it there.
+StreetComplete depends on some projects for the app. Try to find the appropriate place for reporting them.
+
+* **Map style issues** should be reported as in a [separate reporistory](https://github.com/ENT8R/streetcomplete-mapstyle).
```

Comment:
"repository"; "as in" -> "in"; "map, ." -> "map, etc."; "everything, which" -> "everything that"; "markers," -> "markers"; "or an issue" -> "or is an issue"

Reply:
Oh, thanks, it seems I somehow wrote that in a hurry… :smile: